shared/teams/team/rows/bot-row/bot/container.tsx

```diff
@@ -16,6 +18,8 @@ type OwnProps = {
 const blankInfo = Constants.initialMemberInfo
 
 type DispatchProps = {
+  _onEdit: (conversationIDKey: Chat2Types.ConversationIDKey, username: string) => void
+  _onRemove: (conversationIDKey: Chat2Types.ConversationIDKey, username: string) => void
```

Comment:
I thought explicitly typing `DispatchProps` was not needed any more now that mapStateToProps, mapDispatch, mergeProps are inlined in the `connect` call.

Reply:
oh good catch, i'll remove that